stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -1385,6 +1418,59 @@ impl NakamotoChainState {
         Ok(true)
     }
 
+    /// Get the aggregate public key for the given block.
+    pub fn get_aggregate_public_key(
+        sortdb: &SortitionDB,
+        sort_handle: &SortitionHandleConn,
+        chainstate: &mut StacksChainState,
+        header: &NakamotoBlockHeader,
```

Comment:
`header` doesn't really tell me much other than the type of the variable, perhaps, `for_block` ?